packages/vm/tests/BlockchainTestsRunner.js

```diff
@@ -24,11 +26,36 @@ module.exports = async function runBlockchainTest(options, testData, t) {
     validate = true
   }
 
-  const hardfork = options.forkConfigVM
+  let common 
+  
+  if (options.forkConfigTestSuite == "HomesteadToDaoAt5") {
+    // here: get the default fork list of mainnet and only edit the DAO fork block (thus copy the rest of the "default" hardfork settings)
+    const defaultCommon = new Common('mainnet', "dao")
+    // retrieve the hard forks list from defaultCommon...
+    let forks = defaultCommon.hardforks()
+    let editedForks = []
+    // explicitly edit the "dao" block number:
+    for (let fork of forks) {
+      if (fork.name == "dao") {
+        editedForks.push({
+          name: "dao",
+          forkHash: fork.forkHash,
+          block: 5
+        })
+      } else {
+        editedForks.push(fork)
+      }
+    }
+    common = Common.forCustomChain('mainnet', {
+      hardforks: editedForks
+    }, 'homestead') // we should be on the "homestead" fork
```

Comment:
I would love if we can avoid this code block since this is such a long code part for such a super-specialized one case application. Any idea anyone?

If we don't find an optimization I would also here tend to move the functionality (create the custom common instance, so all code within the first `if` clause switch) to a separate function.

Reply:
Maybe we could add functionality to `Common` where we can specify the activation blocks of each fork? Is the `Common` supposed to be static after you instantiate it? Then we could just add a function where we `setHardforkBlock`.